02week/pigLatin.js

```diff
@@ -9,7 +9,22 @@ const rl = readline.createInterface({
 
 
 function pigLatin(word) {
+  const vowels = ['a', 'e', 'i', 'o', 'u']
 
+  const pigLatin = function(arg) {
+    word = arg.trim().toLowerCase().split('');
+    for (let i = 0; i < word.length; i++) {
+// if word starts with a vowel
+      if (vowels.indexOf(word[i]) !== -1 && word[i] === word[0] ) {
```

Comment:
Can you explain the latter part of this if statement, I think it's unnecessary. Why would it matter that the first letter of the word is equal to the current letter?